spec/features/idv/doc_auth/verify_info_step_spec.rb

```diff
@@ -390,4 +390,72 @@
       expect(page).to have_current_path(idv_phone_path)
     end
   end
+
+  context 'with ssn_controller enabled' do
```

Comment:
Is this SSN context block needed here? I think I don't see how it relates.

Reply:
That feature flag is deleted in main, so this file is out of date.